src/workerd/io/actor-cache-test.c++

```diff
@@ -3555,11 +3618,25 @@ KJ_TEST("ActorCache LRU purge ordering") {
 
   // Foo and qux live, bar and baz evicted.
   KJ_ASSERT(KJ_ASSERT_NONNULL(expectCached(test.get("foo"))) == "123");
+  // Note: We are scheduling these to be read from storage.
   (void)expectUncached(test.get("bar"));
   (void)expectUncached(test.get("baz"));
   KJ_ASSERT(KJ_ASSERT_NONNULL(expectCached(test.get("qux"))) == "555");
   KJ_ASSERT(KJ_ASSERT_NONNULL(expectCached(test.get("xxx"))) == "aaa");
   KJ_ASSERT(KJ_ASSERT_NONNULL(expectCached(test.get("yyy"))) == "bbb");
+
+  // Now that we've confirmed `qux`, `xxx`, and `yyy` are in cache, we can let `bar` and `baz` be
+  // inserted into cache. If we did individual `get()`s, then `bar` and `baz` would actually
+  // evict `qux` and `xxx` before we got to verify if they were in cache!
+  mockStorage->expectCall("getMultiple", ws)
```

Comment:
Yes, especially because in some cases we drop it, ex. `ActorCache read overwrite`
```
  // Make some gets but overwrite them in the cache with puts.
  auto promise1 = expectUncached(test.get("foo"));
  auto promise2 = expectUncached(test.get("bar"));
  (void)expectUncached(test.get("baz"));

  test.put("foo", "456");
  test.put("bar", "789");
  test.put("baz", "123");
```

In this case, we don't do the `get()` for `baz`, but in other cases, ex. [this comment](https://github.com/cloudflare/workerd/pull/1814/commits/4a28176ca4d53331308bdfb49d76452d8041d4ca#r1543935758) we still do the get.

It's not yet clear to me what's causing this, but I think it's worth figuring out.